cdap-api/src/main/java/co/cask/cdap/api/mapreduce/MapReduceContext.java

```diff
@@ -144,6 +144,16 @@
   void addOutput(String datasetName, Map<String, String> arguments);
 
   /**
+   * Updates the output configuration of this MapReduce job to also allow writing, based upon the given
```

Comment:
wording a little awkward, what about "allow writing using the given OutputFormatProvider"